src/main/resources/application.yml

```diff
@@ -17,6 +17,19 @@ spring:
         jdbc:
           time_zone: UTC
         format_sql: true
+
+  mail:
+    host: smtp.linchpino.com
+    port: 587
+    username: no-reply@linchpino.com
+    password: NoreplyLinchP!no
```

Comment:
Please assign the host, username, and password as below, as I mentioned twice before:
host: ${MAIL_HOST:smtp.linchpino.com}